plugins/holland.backup.mysql_lvm/holland/backup/mysql_lvm/plugin/common.py

```diff
@@ -19,7 +19,10 @@ def connect_simple(config):
     """
     try:
         mysql_config = build_mysql_config(config)
-        LOG.debug("mysql_config => %r", mysql_config)
+        sanitized = {k:v for k,v in mysql_config.items()}
+        if 'password' in mysql_config.keys():
+            sanitized['password'] = '*' * len(sanitized['password'])
```

Comment:
The length of the password should probably also not be leaked.   Maybe a static length of some small number (`8`?) is better here.

Reply:
Very good point. Replaced the asterisks with just "[REDACTED]"